src/popup/router/components/RecentTransactions.vue

```diff
@@ -139,6 +138,10 @@ export default {
     width: 56px;
     height: 56px;
     margin: 0 auto;
+
+    > path {
+      fill: $color-white;
+    }
```

Comment:
```suggestion
    color: $color-white;
```
won't it work the same?